freud/common.py

```diff
@@ -20,26 +19,22 @@ def convert_array(array, dimensions, dtype=None,
 
     Args:
         array (:class:`numpy.ndarray`): Array to check and convert.
-        dimensions (int): Expected dimensions of the array.
+        dimensions (int): Expected dimensions of the array. If 'None',
+            no dimensionality check will be done (Default value = 'None').
         dtype: code:`dtype` to convert the array to if :code:`array.dtype`
             is different. If `None`, :code:`dtype` will not be changed.
-            (Default value = None).
-        contiguous (bool): Whether to cast the array to a contiguous (Default
-            value = True).
+            (Default value = `numpy.float32`).
         array. Default behavior casts to a contiguous array.
-        array_name (str): Name of the array, used for errors (Default value =
-            None).
 
     Returns:
         py:class:`numpy.ndarray`: Array.
```

Comment:
Probably needs to be `:class:` without the `py:`. Typically this is written as `:py:class:` but we have left off the `:py:` elsewhere in the docs, since it's implied as the default Sphinx domain for Python packages.